docker-entrypoint.sh

```diff
@@ -1,4 +1,11 @@
 #!/bin/sh
 set -e
 privman --init
+
+
+if [ ! -e /usr/local/etc/privoxy/config ] || [ -z "$(ls -A /usr/local/etc/privoxy)" ]; then
+  echo "[entrypoint] void config, populating defaults..."
+  cp -a /opt/privoxy-default/* /usr/local/etc/privoxy/
+fi
```

Comment:
I think this should be documented in the README and warn that changing the configuration may break compatibility with the “privman” script (if not preserving the filterfile and actionsfile options).